components/LobbyCircle.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from '../styles/home.module.scss';
+
+export interface LobbyCircleProps {
+  name: string;
+  // create options
+  // 0 = not create
+  // 1 = create with no lobbies
+  // 2 = create with lobbies
+  create?: number;
```

Comment:
We are using typescript so its a good idea to use it fully!

Check out [enums](https://www.typescriptlang.org/docs/handbook/enums.html)